pybwa/libbwamem.pyx

```diff
@@ -339,8 +335,9 @@ cdef class BwaMemOptions:
     @min_seed_len.setter
     def min_seed_len(self, value: int):
         self._assert_not_finalized(attr_name=BwaMemOptions.min_seed_len.__name__)
-        self._options.min_seed_len = value
-        self._options0.min_seed_len = 1
+        if self._options.min_seed_len != value:
+            self._options.min_seed_len = value
+            self._options0.min_seed_len = 1
```

Comment:
For my own edification - does `_options0` just contain a binary flag for each corresponding member on `_options`, indicating whether they've been modified?

Reply:
That's right!  It really tries to follow [the implementation in bwa](https://github.com/lh3/bwa/blob/b92993c1161e73167181558856567ef2f367e3f0/fastmap.c#L143) so it's easy to verify the translation/porting of any code from bwa to pybwa